pkg/scheduler/framework/plugins/interpodaffinity/scoring.go

```diff
@@ -252,7 +253,8 @@ func (pl *InterPodAffinity) NormalizeScore(ctx context.Context, cycleState *fram
 		return nil
 	}
 
-	var maxCount, minCount int64
+	var minCount int64 = math.MaxInt64
+	var maxCount int64
```

Comment:
we should initialize maxCount to -math.MaxInt64 since anti-affinity scores are in the negative.